core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -166,8 +167,26 @@ private[spark] object JettyUtils extends Logging {
     contextHandler
   }
 
+  private def addDefaultFilters(handlers: Seq[ServletContextHandler], conf: SparkConf): Unit = {
```

Comment:
The problem is we don't have a configuration file for Jetty. So to enable it, we need to set it in the code. Here is an example provided by Jetty: https://github.com/jetty-project/jetty-plugin-support/blob/378f5f691fc24c3f223e7239fc56b3568b6f816e/jetty-servlets/src/test/java/org/eclipse/jetty/servlets/GzipWithPipeliningTest.java#L58